src/util/routerinfo.cc

```diff
@@ -45,15 +48,25 @@ RouterInfoCommand::RouterInfoCommand()
 void RouterInfoCommand::PrintUsage(const std::string& name) const
 {
   LOG(info) << "Syntax: " << name << " routerInfo-(...).dat";
+  LOG(info) << "or: " << name << " --create --host 192.168.1.1 --port 10100 "
+                                 "--floodfill 1 --bandwidth P";
 }
 
 bool RouterInfoCommand::Impl(
     const std::string& cmd_name,
     const std::vector<std::string>& args)
 {
+  bool create = false;
   std::vector<std::string> inputs;
   bpo::options_description options("Specific options");
   options.add_options()(
+      "create,c", bpo::bool_switch(&create)->default_value(false))(
+      "host", bpo::value<std::string>()->default_value("127.0.0.1"))(
+      "port", bpo::value<int>()->default_value(0))(
+      "netid", bpo::value<int>()->default_value(I2P_NETWORK_ID))(
```

Comment:
I thought that maybe it can ensure that a private testnet doesn't mix with the public network.  Also, maybe, one day, we will want to have a public testnet (still don't know if that would really be usefull).